libs/gl-client-py/tests/test_plugin.py

```diff
@@ -30,4 +30,10 @@ def test_max_message_size(clients):
     
     # Send message too large.
     with pytest.raises(ValueError):
-        n1.datastore("some-key", hex=bytes.fromhex(secrets.token_hex(size)))
\ No newline at end of file
+        n1.datastore("some-key", hex=bytes.fromhex(secrets.token_hex(size)))
+
+def test_node_info(clients):
+    c1 = clients.new()
+    c1.register()
+    n1 = c1.node()
+    print(n1.get_info())
```

Comment:
Aren't there already several tests that do this and more? While I share the desire to have a test have a single responsibility that is not possible for slow integration tests.